tree/dataframe/src/RMetaData.cxx

```diff
@@ -36,77 +36,109 @@ RMetaData &RMetaData::operator=(RMetaData &&) = default;
 
 RMetaData::~RMetaData() = default;
 
+/// @brief Add an RMetaData class instance.
+/// @param key input key for a given RMetaData instance.
```

Comment:
This is very nitpicky, but:
```suggestion
/// @param[in] key input key for a given RMetaData instance.
```
(taken from https://root.cern/for_developers/doxygen/#method-documentation, also applies to the subsequent docstrings)

Reply:
thanks! I missed it, definitely, we should be consistent